packages/mdc-button/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@material/elevation": "^0.28.0",
     "@material/ripple": "^0.32.0",
+    "@material/rtl": "^0.32.0",
```

Comment:
Have you tried `npm install`ing this? I think the latest version of mdc-rtl is still 0.30.0 due to no changes since then.

Reply:
I have not tried it, like i have not tried gazillion other things I could have tried... But I should have checked what rtl version there is. it is 30 not 32.